annotate.py

```diff
@@ -25,54 +30,47 @@ def draw_circle(event,x,y,flags,param):
         if event == cv2.EVENT_LBUTTONDOWN:
             drawing = True
             ix,iy = x,y
-            cv2.circle(img, (x, y), small_size,RED,-1)
+            cv2.circle(img, (x, y), small_size,BLUE,-1)
             print(ix, "x  ", iy,"y")
-
-        elif event == cv2.EVENT_MOUSEMOVE:
-            if drawing == True:
-                cv2.circle(img, (x, y), small_size,RED,-1)
-                
+          
         elif event == cv2.EVENT_LBUTTONUP:
             drawing = False
+ 
 
         if event == cv2.EVENT_MBUTTONDOWN:
             rdrawing = True
             ix,iy = x,y
-            cv2.circle(img, (x, y), large_size,BLUE,-1)
+            cv2.circle(img, (x, y), large_size,RED,-1)
 
             print(ix, "x", iy,"y")
 
-        elif event == cv2.EVENT_MOUSEMOVE:
-            if rdrawing == True:
-                cv2.circle(img, (x, y), large_size,BLUE,-1)
-
         elif event == cv2.EVENT_MBUTTONUP:
             rdrawing = False
+                     
+
 
         
 ##Set up for a single image
-#function: annotate_img
-# Inputs
-#    img_path - name and location of the image, example "B.jpg"
-#    size1 - size of the 1st brush circle, in px
-#    size2 - size of the 2nd brush circle, in px
-#    initials - First name and last name initials of the person doing the labelling. Recorded next to every PMT feature ID line in the text file.
-# Outputs
-#    {image_name}.png - A map of all the features in binary - features labelled in blue are recorded as RGB (1,1,1), and features in red are recorded as RGB (2,2,2).
-#    {image_name}.txt - A text file. Each line contains a PMT feature ID and its pixel coordinates on the image. The text file has the same name as the image.
-
-##User keypresses (make sure the image window is selected):
-# click on image to grab pixel coordinates:
-#	 Left-click to label with red 
-#	 middle-click to label with blue
-# s to close image and exit program
-# r to write coordinates (you will be prompted for the PMT feature ID if a PMT has not been selected yet)
-# f to select new PMT
-# n to increment the featureID number
-# b to decrement the featureID number
+##function: annotate_img
+    ##inputs
+        ##img_path - name and location of the image, example "B.jpg"
+        ##size1 - size of the 1st brush circle, in px
+        ##size2 - size of the 2nd brush circle, in px
+        ##initials - First name and last name initials of the person doing the labelling. Recorded next to every PMT feature ID line in the text file.
+    ##outputs
+        ##label.jpg - A map of all the features in binary (white on black)
```

Comment:
Seems like older version of documentation got injected here?

Reply:
Hmm yeah can't remember how that happened lol
Sorry about that I'll make a new commit